xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/store/migration/hibernate/R140600000XWIKI19869DataMigration.java

```diff
@@ -271,7 +271,7 @@ private void handleHistory(XWikiDocument userDoc) throws XWikiException
      * @param resetPassword {@code true} if the password should be just reset, or {@code false} if the hash of the
      *                      defined password should be computed.
      */
-    private boolean fixPasswordHash(XWikiDocument userDoc, boolean isMain, boolean resetPassword)
+    private boolean fixPasswordHash(XWikiDocument userDoc, boolean isMain, boolean resetPassword) throws XWikiException
```

Comment:
It seems a bit strange to expect for this method to potentially fail. My understanding is that it cannot really happen and if it did, we would be quite stuck.